chain/chain_config.go

```diff
@@ -89,6 +89,8 @@ type Config struct {
 	ForkID6IncaBerryBlock   *big.Int `json:"forkID6IncaBerryBlock,omitempty"`
 	ForkID7EtrogBlock       *big.Int `json:"forkID7EtrogBlock,omitempty"`
 	ForkID88ElderberryBlock *big.Int `json:"forkID88ElderberryBlock,omitempty"`
+
+	SupportZeroGas bool `json:"supportZeroGas,omitempty"`
```

Comment:
I was wondering for a while, what if we have ZkConfig with all these fields and then just combine it as an anonymous field to Config?

like 

```
type ZkConfig struct {
     ForkID4Block ...
     ...
}

type Config struct {
      ZkConfig
      ....
 }
 ```
 
 
 will be an easier rebase